### PR TITLE
VIH-10001 utilise the SupportsWorkAllocation property to correctly assign AllocatedTo 

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-10001

### Change description ###

provide allocated cso name or not required if venue does not suppport work allocation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/Mappings/ConferenceForVhOfficerResponseMapper.cs [hunk 1]
```diff
@@ -19,6 +19,16 @@ public ConferenceForVhOfficerResponseMapper(IMapTo<IEnumerable<ParticipantSummar
...
         public ConferenceForVhOfficerResponse Map(ConferenceForAdminResponse conference, AllocatedCsoResponse allocatedCsoResponse)
         {
+            string allocatedCso;
+            if(!allocatedCsoResponse?.SupportsWorkAllocation ?? false)
+            {
+                allocatedCso = "Not Required";
+            }
+            else
+            {
+                allocatedCso = allocatedCsoResponse?.Cso?.FullName ?? "Unallocated";
```
Comment (marcogagliardi) [inline VideoWeb/VideoWeb/Mappings/ConferenceForVhOfficerResponseMapper.cs:29]: should it be Not Allocated as in Booking Api

**Review by marcogagliardi — COMMENTED** [review]
do we need to add the env variable in values.dev.template.yaml?

**Review by marcogagliardi — APPROVED** [review]
👌🏽

**Review by phildebrah — APPROVED** [review]
Tested and Approved

**Review by oliver-scott — APPROVED** [review]

**Review by phildebrah — APPROVED** [review]
Approved

**Pull request merged**